JavaAssignment/src/kunal/PerimeterArea.java

```diff
@@ -0,0 +1,35 @@
+package kunal;
+public class PerimeterArea 
+{
+    int radius, length, breadth;
+    double pi = 3.14;
+	double perimeter;
+	double Area;
+	double Area1;
+   
+    void circle(int radius)
+    {
+        perimeter = 2 * pi * radius;
+		Area1= pi * radius * radius;
+        System.out.println("Perimeter of circle:"+perimeter);
+		System.out.println("Area of circle:"+Area1);
+    } 
+    
+    void rectangle(int length,int breadth)
```

Comment:
if you are finding an area of rectangle write like "areaOfRectangle" as a method name. And comment before this method.